### DISCUSSION TITLE
Are multi page PDFs supported for chatting over? I am getting errors

### DISCUSSION THREAD

**Discussion by elabz** [open-webui/open-webui#1095] [General]
Hello y'all, what a wonderful project! If only I can figure out how to load PDFs to chat over them.

I have this very simple PDF, but it's multipage (IC spec), is this something that's supported? Apparently, I was able to load one-page one
[LM1296_NationalSemiconductor.pdf](https://github.com/open-webui/open-webui/files/14532134/LM1296_NationalSemiconductor.pdf)
<img width="868" alt="Screenshot 2024-03-07 at 8 40 47 PM" src="https://github.com/open-webui/open-webui/assets/9451173/dd10d09e-dfd8-43c2-a540-e386a4db6a99">

This is the error message I am getting if it helps:

```
INFO:     192.168.0.191:49673 - "GET /api/v1/documents/ HTTP/1.1" 200 OK
application/pdf
cannot reshape array of size 4522 into shape (1096,1581,newaxis)
INFO:     192.168.0.191:49675 - "POST /rag/api/v1/doc HTTP/1.1" 400 Bad Request
INFO:     192.168.0.191:50082 - "GET /api/v1/documents/ HTTP/1.1" 200 OK
application/pdf
cannot reshape array of size 4522 into shape (1096,1581,newaxis)
INFO:     192.168.0.191:50084 - "POST /rag/api/v1/doc HTTP/1.1" 400 Bad Request

```

Thanks for all your input y'all 🤗 !

**Comment by tjbck** [comment]
@jannikstdl Seems like an OCR error, could you share your pdf file so we can narrow down the issue?

**Comment by jannikstdl** [comment]
Hi, can you rename the PDF File and try it again?

**Reply by elabz** [reply]
@jannikstdl , is there a particular way I should rename it? The file name as it is seems to be ... unremarkable? Is there a better way to name those files?

**Reply by elabz** [reply]
Renaming the file to simply `renamed.pdf` and re-uploading it did not change anything - still the same error. 
@tjbck , the PDF in question is attached at the top of the thread. Thank you for looking into this!

**Reply by jannikstdl** [reply]
Ok thanks. It was a try to rule out an error. Will investigate further.

**Reply by jannikstdl** [reply]
Are you using the default embedding model?

**Reply by elabz** [reply]
@jannikstdl , I did not set one up, although I may have nomic downloaded on one of the installs (I tried it on 2). So, I guess, whichever it pics by default? In fact, didn't know that there may be a choice of embeddings given for RAG, will go look it up, thanks!

**Reply by jcrosasm-IA** [reply]
Hi, OpenWebUI, definitely the best project of this type I've tried so far, however, I also get an error when I have OCR enabled and load a pdf document with text and image. :(

**Reply by cpolcino** [reply]
ECSS-Q-ST-80C(6March2009).pdf, same problem with this file(copy the name and you find it on google)

**Reply by r00t1ng** [reply]
Is there an update here? I have similar issues
